UsingSeal.md

```diff
@@ -191,28 +191,38 @@ Currently, access to above key servers is permissive. However, rate limiting is
 
-Key servers can be registered onchain to enable discoverability. To register a key server, call the `create_and_transfer_v1` function in the `seal::key_server` module. For example:
+### Open mode
+In this mode Seal policies of any package can be queried.
+This is an ideal option for testing Seal, or if the key server does not have any direct liabilty to users (*best-effort* service).
+
+First, a master BLS keypair should be generated.
+Use the `seal-cli` tool for that using `cargo run --bin seal-cli genkey`.
+It outputs the master secret key and the public key.
+
+Key servers should be registered onchain to enable discoverability.
+To register a key server, call the `create_and_transfer_v1` function in the `seal::key_server` module. For example:
 
 ```shell
 sui client call --function create_and_transfer_v1 --module key_server --package 0x62c79dfeb0a2ca8c308a56bde530ccf3846535e1623949d45c90d23128afff52 --args mysten-dev-1 https://seal-key-server-testnet-1.mystenlabs.com 0 0xa023acbf600401017ee17bf918106ea9911914ca017aa3ab9ab5c64beb9bb5236fd9d4d5b5645dc3bc0d4f732ed04fc60d14b9f37987fe5eeb4db07fc0982904ce1ed0b07607ae2e99086e141f6c6a1df6def5f5d434ca7c09856a3750c92969 --gas-budget 10000000
 ```
 
-Run the server using `cargo run --bin key-server` either with the environment variable `CONFIG_PATH` pointing to a .yaml configuration file (see [crates/key-server/key-server-config.yaml] for an example) or by setting the required environment variables directly:
-- `MASTER_KEY` is the master secret key generated by the `seal-cli` tool.
-- `KEY_SERVER_OBJECT_ID` is the object id of the registered key server.
-- `NETWORK` specifies the network to connect to, such as `testnet`, `mainnet`, or other supported environments. To use a custom full node, set `NETWORK` to `custom`. When using the `custom` option, you must also set the `NODE_URL` environment variable with the URL of your Full node endpoint.
+Run the server using `cargo run --bin key-server` with the environment variable 
+`MASTER_KEY` set to the master secret key generated by the `seal-cli` tool,
+and the environment variable `CONFIG_PATH` pointing to a .yaml configuration file. See [example](crates/key-server/key-server-config.yaml) for the format of the configuration file and use the `Open` mode. Set `key_server_object_id` to the id of the registered key server object.
 
 Example:
 ```shell
-export MASTER_KEY="KYinoC5hVWeWqOUU9dw7PVHiROYFWB/nQZ55Kmytjig="
-export KEY_SERVER_OBJECT_ID="0x1ee708e0d09c31593a60bee444f8f36a5a3ce66f1409a9dfb12eb11ab254b06b"
-export NETWORK="testnet"
+export MASTER_KEY="0x652081a0ad50cd516e8cb82e44653793a27853350d35d724fded523e85ed0382"
+export CONFIG_PATH="/Users/alice/seal/key-server-config.yaml"
 cargo run --bin key-server
 ```
 
+// TODO: update the docker
```

Comment:
What does this refer to? Does it need to be updated before being merged?

Reply:
it's a todo to fix the docker